specs/Standards/Sould Bound Token/README.md

```diff
@@ -0,0 +1,454 @@
+    deactivate SBT2
+
+    Alice-->>SBT_Registry: sbt(SBT_2_Contract, 7991)
+    SBT_Registry-->>Alice: {token: 7991, owner: alice2, metadata: metadata2}
+
+    Note over Alice,SBT_Registry: After a while Alice decides to merge her `alice2` account into `alice2`
+    Alice->>SBT_Registry: sbt_soul_transfer(alice) --accountId alice2
+
+    Alice-->>+SBT_Registry: sbt_tokens_by_owner(alice2)
+    SBT_Registry-->>-Alice: []
+
+    Alice-->>+SBT_Registry: sbt_tokens_by_owner(alice)
+    SBT_Registry-->>-Alice: [[SBT_1_Contract, [238]], [SBT_2_Contract, [7991, 7992]]]}
+```
+
+## Consequences
+
+### Positive
+
+- Template and set of guidelines for creating SBT tokens.
+- Ability to create SBT aggregators.
+- Ability to use SBT as a primitive to model non KYC identity, badges, certificates etc...
+- SBT can be further used for "lego" protocols, like: Proof of Humanity (discussed for NDC Governance), undercollateralized lending, role based authentication systems, innovative economic and social applications...
+- Standard recoverability mechanism.
+- SBT are considered as a basic primitive for Decentralized Societies.
+- new way to implement Sybil attack resistance.
+
+### Neutral
+
+- The API partially follows the NEP-171 (NFT) standard. The proposed design is to have native SBT API and also support NFT based queries.
+  NOTE: we can decide to use `nft_` prefix whenever possible.
+
+### Negative
+
+- new set of events to be handled by the indexer and wallets.
+- complexity of integration with a registry: all SBT related transactions must go through Registry.
+
+## Considerations
+
+Being fully compatible with NFT standard is a desirable. However, given the requirements related to _soul transfer_ we didn't find an applaudable solution. Also we decided to use u64 as a Token ID, diverging further from the NFT NEP-171 standard.
+
+Give that our requirements are much striker, we need to reconsider the level of compatibility with NEP-171 NFT.
+There are so many examples where NFT standards are poorly or improperly implemented, adding another standard with differing functionality but equal naming in there will cause lots of misclassifications between NFTs/SBTs, and then recover methods called on NFT contracts, SBTs attempted to be listed on NFT marketplaces and so on.
+
+CALL FOR ACTION: Shall the events be issued by the registry or by the issuing contract?
+
+- registry: we query registry, so it makes sense to use registry. If we emit the event by the registry, we need to add smart contract argument as a field of the events.
+- maybe we should reduce amount of tokens: merge {mint, renew}?
+- Should we use NEP-171 Mint and NEP-171 Burn (instead of revoke) events? If the events will be emitted by registry, then we need new events to include the contract address. Since the native token ID is different in SBT and NFT, we should probably keep SBT events version. Also, with registry, it makes less sense to issue NEP-171 Mint and Burn.
```

Comment:
SBT-specific events are more flexible even for future extension; keep them and avoid any temptation to abuse NFT infrastructure. I won't approve a NEP that partially overlaps with the interfaces of some other NEP as it creates an unnecessary hassle for the ecosystem.

Reply:
Agree, I have the same preference. However posting it here to open a discussion.
In fact, we were discussing is to use these events in MT, as they reflect the needs there as well. Let's think about it more.
cc @uncle-T0ny, @marco-sundsk